var/spack/repos/builtin/packages/py-morph-tool/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMorphTool(PythonPackage):
+    """Python morphology manipulation toolkit"""
+
+    homepage = "https://github.com/BlueBrain/morph-tool"
+    git = "https://github.com/BlueBrain/morph-tool.git"
+    pypi = "morph-tool/morph-tool-2.9.1.tar.gz"
+
+    version("master", branch="master")
+    version("2.9.1", sha256="305e9456c8047726588b23dfa070eb95ccbe5573e9fea3e0a83dc93eacdf61dc")
+    version("2.9.0", sha256="c60d4010e17ddcc3f53c864c374fffee05713c8f8fd2ba4eed7706041ce1fa47")
+
+    variant("neuron", default=False, description="Enable additional neuron support")
+    variant("plotly", default=False, description="Enable additional plotly support")
```

Comment:
```suggestion
    variant("nrn", default=False, description="Enable additional neuron support")
    variant("plot", default=False, description="Enable additional plotly support")
```
To match extra names

Reply:
not sure if understood `extra` names part here but we could keep the name `neuron` as it's the familiar to the users/existing deployments via different forks?

Reply:
`plotly` at least didn't exist before, so that one shouldn't be an issue to change. 

@adamjstewart  would you be opposed to keeping `neuron` as per @pramodk 's argument?

Reply:
I would rather match the behavior of pip than the behavior of some random fork of Spack. By "extra", I mean the name of a collection of optional dependencies specified using `extras_require` in setuptools (same as Spack's "variants"). In this example, users normally use:
```console
> pip install morph-tool[nrn]
```
to install morph-tool with the optional neuron dep(s). Similarly, it would make sense if Spack did this using:
```console
> spack install py-morph-tool+nrn
```